tools/site_and_regional/run_neon.py

```diff
@@ -150,99 +138,52 @@ def get_parser(args, description, valid_neon_sites):
                 ''', 
                 required = False,
-                type = int, 
-                default = 100)
+                type = str, 
+                default = '0Y')
 
-    tr_parser.add_argument('--start-year',
+    parser.add_argument('--start-date',
                 help='''           
-                Start year for running CTSM simulation.
+                Start date for running CTSM simulation in ISO format.
                 [default: %(default)s]
                 ''',               
                 action="store", 
-                dest="start_year",                                                                                                                                                 
+                dest="start_date",                                                                                     
                 required = False,
-                type = int,        
-                default = 2018)
+                type = datetime.date.fromisoformat,
+                default = datetime.datetime.strptime("2018-01-01",'%Y-%m-%d'))
 
-    tr_parser.add_argument('--end-year',
+    parser.add_argument('--end-date',
                 help='''
-                End year for running CTSM simulation.
+                End date for running CTSM simulation in ISO format.
                 [default: %(default)s]
                 ''', 
                 action="store",
-                dest="end_year",
+                dest="end_date",
                 required = False,
-                type = int,
-                default = 2020)
-
-    #parser.add_argument('--spinup',
-    #            help='''
-    #            AD spin-up
-    #            [default: %(default)s]
-    #            ''', 
-    #            action="store_true",
-    #            dest="ad_flag",
-    #            required = False,
-    #            default = True)
-    #parser.add_argument('--postad',
-    #            help='''
-    #            Post-AD spin-up
-    #            [default: %(default)s]
-    #            ''', 
-    #            action="store_true",
-    #            dest="postad_flag",
-    #            required = False,
-    #            default = True)
-    #parser.add_argument('--transient',
-    #            help='''
-    #            Transient
-    #            [default: %(default)s]
-    #            ''', 
-    #            action="store_true",
-    #            dest="transient_flag",
-    #            required = False,
-    #            default = True)
-
-    #parser.add_argument('--sasu','--matrix',
-    #            help='''
-    #            Matrix (SASU) spin-up
-    #            [default: %(default)s]
-    #            ''', 
-    #            action="store_true",
-    #            dest="sasu_flag",
-    #            required = False,
-    #            default = False)
+                type = datetime.date.fromisoformat,
+                default = datetime.datetime.strptime("2020-01-01",'%Y-%m-%d'))
```

Comment:
I guess I'd assume that the end date for ad spinup and postad should default to 2021-01-01, unless there are missing data in 2020?  Will this work with leap years correctly?

Reply:
Right - I've changed this to 2021-01-01.  The model doesn't know about leap years.